WcaOnRails/lib/paypal_interface.rb

```diff
@@ -96,11 +140,13 @@ def self.capture_payment(competition, order_id)
 
       # Logs requests and responses.
       # By default, it only logs the request method and URL, and the request/response headers.
-      builder.response :logger
+      builder.response :logger, ::Logger.new($stdout), bodies: true if Rails.env.development?
     end
   end
 
   private_class_method def self.generate_access_token
+    return '' if EnvConfig.SKIP_PAYPAL_AUTH?
```

Comment:
Heavens no. Big fat veto. What are you trying to accomplish here and why does `Rails.env.test?` not cut it?

Reply:
The issue I was having is the competition controller trying to call the auth endpoint to generate the onboarding URL (line 319), while the auth endpoint had been mocked in a different test - so I would SKIP_PAYPAL_AUTH locally only, when I was only running a small subset of tests. 

Another viable option here would be to disable the stub 
- as part of test cleanup after it was created (best solution actually) 
- around the tests which call the Paypal onboarding URL

I'm happy to update the PR to disable the stub, or include that change in a subsequent PR if we just want to focus on getting this one merged for now.

Reply:
(reasoning here is important so leaving you to Resolve the conversation)